### ISSUE TITLE
[IMPROVEMENT] Facts & Figures

### ISSUE THREAD

**Issue by RobZenFL** [FLBadoras/wiki#101]
### Grobbeschreibung

Man könnte noch eine Seite einbauen, auf der gewisse Zahlen abrufbar sind. Ich denke da an interessante Facts wie Anzahl der Scripts, Codezeilen, Quest-Optionen, Quests, Anzahl NPCs, usw usf.

Man müsste dafür sicherlich mal zusammentragen, was es für interessante Daten gibt und ob das ausreicht, um eine eigene Seite dafür zu gestalten.

### Beschreibung

s.o.

**Comment by motrellin** [comment]
Könnte man. Aber ist das nicht eher interessanter Social Media oder Werbungscontent, der potenziell besser auf der Website verortet ist?

**Comment by RobZenFL** [comment]
Ja, hab das im Mini-Format auf der Seite, will dafür aber das Layout nicht nochmal aufreißen, nur um da mehr reinzuschreiben.
Kann man als Idee ja mal behalten, ist halt tiefergehend als auf der Seite, so wie das ganze Wiki. Dachte deswegen passts vllt ganz gut.

**Comment by motrellin** [comment]
Ja okay. Kann man ja als einfache Tabelle oder so machen, also feel free.
